src/components/common/Extensions/ImageCarousel/ImageCarousel.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC } from 'react';
+import { Carousel } from 'react-responsive-carousel';
+import { images } from './const';
+import styles from './ImageCarousel.module.css';
+import { ImageCarouselProps } from './ImageCarousel.types';
+
+const displayName = 'common.Extensions.ImageCarousel';
+
+const ImageCarousel: FC<ImageCarouselProps> = ({ slidePercentage = 95, transitionTime = 300, slideUrls = images }) => {
+  return (
+    <div className={styles.carouselWrapper}>
+      <Carousel
```

Comment:
Is it rendered as UL in HTML? If not, please change the `LI` elements on `DIV` to keep the semantic code.